Frontend/ui-customer/index.js

```
@@ -445,21 +462,30 @@ window["onEvent"] = function (jsonPayload, args, callback) { // onEvent from hyp
+} else {
+  console.error("Calling action DUI_READY");
+  if (window.JBridge.emptyCallbackQueue)
+    window.JBridge.emptyCallbackQueue()
+    JBridge.runInJuspayBrowser("onEvent", JSON.stringify({
+      action: "DUI_READY",
+      event: "initiate",
+      service: JOS.self
+    }), "");
 
-eval(window.JBridge.loadFileInDUI("v1-tracker.jsa"));
+    eval(window.JBridge.loadFileInDUI("v1-tracker.jsa"));
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)